### ISSUE TITLE
Is no-brace on enemy contact still intentional

### ISSUE THREAD

**Issue by IceRaptor** [BattletechModders/CBTBehaviorsEnhanced#43]
From rogueticket-230: 

No  brace on a jump In non-interleaved mode
In the last non-combat turn, in which the enemy was noticed, and after which the combat begins, there is no brace after jump.
The second typical case - after the end of the interleaved mode, when the non-combat mode is launched - no bracing after jumping.
Interactive Rubber Dolphin05/09/2020
yes, I believe the brace only happens when no enemy is detected
I think that is intentional
@FrostRaptor ?
Grey W05/09/2020
But what's the point? Suppose we destroyed one group of opponents, the battle is over and we are moving to the next group. Why is there no brace after contact with the first group of the enemy?
https://youtu.be/RXUIQK1ryCQ
YouTube
GreyW3
No brace in non interleaved mode

FrostRaptor05/09/2020
This is intentional, yes. It's to allow use of sensor lock and other abilities during the non-interleaved turn.
Basically, flexibility as you encounter the enemy.
That's the reason I was given when I was asked to make it this way.
Interactive Rubber Dolphin05/09/2020
but the turn autoends?
you don't get to do anything
am I missing something
FrostRaptor05/09/2020
Hurm
When you're in non-interleaved and there's an enemy present, it auto-ends?
Interactive Rubber Dolphin05/09/2020
yes
nothing changes I belive
Grey W05/09/2020
yes it auto ends
FrostRaptor05/09/2020
Hurm
Alright, I'll look at it.
I thought it was intentional. If it's not, I'll fix it.
Sorry for the hassle. Thanks for the report.
Grey W05/09/2020
glad to help

**Comment by IceRaptor** [comment]
The assertion is that mechs cannot sensor lock or do other actions in non-interleaved mode at present, so why not auto-brace them when they move during the contact phase? Need to research this and validate it, but seems like a sensible suggestion.

**Comment by IceRaptor** [comment]
Implemented a long time ago - fixed
